lib/order.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :take_order, :menu
+
+  def initialize(menu = Menu.new)
+    @menu = menu
+    @take_order = []
+  end
+
+  def customer_order(item, amount)
+    @take_order << { item: item, amount: amount }
+  end
+
+  def total_cost
+    total = 0
+
+    @take_order.each do |food_item|
+      total += @menu.menu_items[food_item[:item]] * food_item[:amount]
```

Comment:
This bombs out when the user enters a string - consider a `to_sym` when taking customer orders (which will probably be a string if it comes via `gets` for example. The customer doesn't know to use a method otherwise.